### DISCUSSION TITLE
Kong gateway 3.3 on ARM

### DISCUSSION THREAD

**Discussion by andy6812345** [Kong/kong#13077] [Help] [answered]
Hi,
Error encountered during installation of Kong gateway 3.3 on ARM architecture server.
error msg:软件包 kong-enterprise-edition-0:3.3.1.1-1.x86_64是针对different架构

Thanks

**Comment by StarlightIbuki** [comment, marked as answer]
https://hub.docker.com/r/arm64v8/kong/
